### PR TITLE
Adding Sticker Mule link in the about page

### PR DESCRIPTION
**Pull request by anicolaspp**
The link points to the Typelevel sticker mule store

### CODE REVIEW

File: css/main.scss [hunk 1]
```diff
@@ -808,6 +808,10 @@ blockquote {
   background-image: url('../img/ico-email.svg');
   background-size: 32px 32px;
 }
+.contact__item--Sticker-mule {
+  background-image: url('../img/ico-smule.png');
+  background-size: 80px 80px;
```
Comment (larsrh) [inline css/main.scss]: Please make this 32x32 as the others.

File: about.html [hunk 2]
```diff
@@ -20,6 +20,15 @@
     <a href="mailto:{{site.email}}" class="contact__item contact__item--email">
       <h4>Email</h4>
     </a>
+
+  </div>
+
+  <h2> Stickers! </h2>
```
Comment (larsrh) [inline about.html]: Could you please replace "Stickers" by "Merchandise" and add an additional item for our [Spreadshirt shop](https://shop.spreadshirt.co.uk/typelevel/)?

File: css/main.scss (hunk 1 above)
Comment (anicolaspp) [inline css/main.scss]: if I change this to 32x32 it will not show nicely. Ideas?

File: about.html (hunk 2 above)
Comment (anicolaspp) [inline about.html]: I am going to need a logo for the Spreadshirt shop

**Comment by larsrh** [comment]
Excellent! Looks good from my end.

**Pull request merged**